lib/unleash_context.dart

```diff
@@ -29,4 +29,22 @@ class UnleashContext {
 
     return params;
   }
+
+  Map<String, String> toFlatMap() {
```

Comment:
nested properties from toMap are sent to the backend. toFlatMap is used to create a structure that matches setContextField format that is always flat e.g. `setContextField('myField', 'myValue')`

Reply:
Added propert eq and hashCode instead